Samples/MobileBuyIntegration/MobileBuyIntegration/SettingsViewController.swift

```diff
@@ -154,11 +154,6 @@ class SettingsViewController: UITableViewController {
 		case Section.colorScheme:
 			let newColorScheme = colorScheme(at: indexPath)
 			ShopifyCheckout.configuration.colorScheme = newColorScheme
-			let navigationBarAppearance = newColorScheme.navigationBarAppearance
-			UINavigationBar.appearance().standardAppearance = navigationBarAppearance
-			UINavigationBar.appearance().scrollEdgeAppearance = navigationBarAppearance
-			navigationController?.navigationBar.standardAppearance = navigationBarAppearance
-			navigationController?.navigationBar.scrollEdgeAppearance = navigationBarAppearance
```

Comment:
This fixes the issue of the border separation being present between the header and the webview content when the theme settings changes